coordinator/src/main/scala/com/socrata/datacoordinator/service/Mutator.scala

```diff
@@ -268,6 +272,8 @@ class Mutator[CT, CV](indexedTempFile: IndexedTempFile, common: MutatorCommon[CT
             val updateOnly = getWithStrictDefault[Boolean]("update_only", false)
             val bySystemId = getWithStrictDefault[Boolean]("by_system_id", false)
             RowData(index, truncate, mergeReplace, nonFatalRowErrorsClasses, updateOnly = updateOnly, bySystemId = bySystemId)
+          case NoOp =>
+            NoOpCommand
```

Comment:
Pass the `index` into the command here.  It doesn't look like it's needed for anything, but it's best to keep the pattern constant.

What on earth is is this for though, and how does it relate to the PR description?

Reply:
There will be metadata added to pg secondary to control indexing behavior.  Initially it is going to be manually entered.  In the future, it will be driven by some other automatic mechanism. We are using the same secondary replication version update mechanism to drive index update or rebuild (like after we mess around with secondary indexes)

Reply:
Ah.  You don't actually need a no-op command for that, then.  If all you want is to poke secondaries, a mutation script with a "normal" command header and no subsequent commands will create a new data version with an empty update list that gets pushed out to secondaries.

Reply:
ok, i'll try that